layouts/_default/single.html

```diff
@@ -49,6 +49,8 @@ <h2>
   {{ partial "posts_pagination.html" . }}
   {{ end }}
 
+  {{ if eq (.Params.hideComments | default false) false }}
   {{ partial "comments.html" . }}
+  {{ end }}
```

Comment:
I think this should also work (but I'm not 100% sure):

```suggestion
  {{ if not (.Params.hideComments | default false) }}
    {{ partial "comments.html" . }}
  {{ end }}
```

can you check this?

Reply:
Whenever we deliver a new flag, we should update a post archetype (https://github.com/panr/hugo-theme-terminal/blob/master/archetypes/posts.md) with the default value as well.

Reply:
You are right that the above code works as well. I've added the default to the archetype as well 👍